Python/rerf/rerfClassifier.py

```diff
@@ -73,6 +76,21 @@ class rerfClassifier(BaseEstimator, ClassifierMixin):
         ``None`` means 1. ``-1`` means use all processors. 
     random_state : int or None, optional (default=None)
         Random seed to use. If None, set seed to ``np.random.randint(1, 1000000)``.
+    
+    image_height : int, optional (default=None)
+        S-RerF required parameter. Image height of each observation.
+    image_width : int, optional (default=None)
+        S-RerF required parameter. Width of each observation.
+    patch_height_max : int, optional (default=None)
+        S-RerF parameter. Maximum image patch height to randomly select from.
+        If None, set to ``image_height``.
+    patch_height_min : int, optional (default=1)
+        S-RerF parameter. Minimum image patch height to randomly select from.
+    patch_width_max : int, optional (default=None)
```

Comment:
i was thinking about this. maybe default should be sqrt of image height/width for image width/height instead of max height/width?

Reply:
I like that much better than the full extent.

Reply:
Done